src/index.js

```diff
@@ -0,0 +1,157 @@
+//////////////temperature section/////////////////
+
+// Select temperature
+const currentTemp = document.getElementById('current-temp');
+
+//select weather garden bottom picture element
+const weatherGardenChanger = document.getElementById('bottom-landscape');
+
+//function to change background of the picture
+const landscapeChanger = (temp) => {
+  if (temp > 26) {
+    weatherGardenChanger.innerHTML = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (temp > 21) {
+    weatherGardenChanger.innerHTML = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (temp > 15) {
+    weatherGardenChanger.innerHTML = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else {
+    weatherGardenChanger.innerHTML = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
+
+// Function to increment temperature
+const handleTempIncrement = () => {
+  let temperature = currentTemp.innerHTML;
+  temperature++;
+  currentTemp.innerHTML = temperature;
+  changeTempColor(temperature);
+  landscapeChanger(temperature);
+};
+// Function to decrement temp
+const handleTempDecrement = () => {
+  let temperature = currentTemp.innerHTML;
+  temperature--;
+  currentTemp.innerHTML = temperature;
+  changeTempColor(temperature);
+  landscapeChanger(temperature);
+};
+// Select increment and decrement buttons
+const incrementTemp = document.getElementById('increment-count');
+const decrementTemp = document.getElementById('decrement-count');
+
+// Add click event to buttons
+incrementTemp.addEventListener('click', handleTempIncrement);
+decrementTemp.addEventListener('click', handleTempDecrement);
+
+//change color if the temp increases
+const changeTempColor = (temp) => {
+  if (temp > 26) {
+    currentTemp.style.color = 'red';
```

Comment:
Style: It would better follow the [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns) principle if this set a class (like `hot`) instead of directly altering the styles.